chatops-lark/README.md

```diff
@@ -8,6 +8,7 @@ You can run it by following steps:
   ```yaml
   cherry-pick-invite.audit_webhook: <your_audit_lark_webhook>
   cherry-pick-invite.github_token: <your_github_token>
+  bot_name: <your_bot_name>  # bot name in lark
```

Comment:
Can we make the bot get is's name once in staring?

Reply:
Good idea, I'll try to see if I can get it.

Reply:
api ref: https://open.feishu.cn/document/client-docs/bot-v3/obtain-bot-info